templates/myaccount/subscription-totals-table.php

```diff
@@ -76,7 +76,7 @@
 				<?php
 			}
 
-			$purchase_note = get_post_meta( $_product->get_id(), '_purchase_note', true );
+			$purchase_note = $_product->get_meta( '_purchase_note', true );
```

Comment:
Can use the proper getter here:
```suggestion
			$purchase_note = $_product->get_purchase_note();
```